meson.build

```diff
@@ -8,7 +8,7 @@ project('nnstreamer', 'c', 'cpp',
   default_options: [
     'werror=true',
     'warning_level=1',
-    'c_std=gnu89',
+    'c_std=c89',
```

Comment:
have to fix error after changing this,
error: implicit declaration of function ‘fileno’

Reply:
I'll wait for ORC update before proceeding with this PR.